ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -87,15 +87,16 @@ queryEncodeNodeToClient ::
   -> Encoding
 queryEncodeNodeToClient codecConfig queryVersion blockVersion (SomeSecond query)
   = case queryVersion of
-    TopLevelQueryDisabled -> case query of
-      BlockQuery blockQuery -> encodeNodeToClient
-        @blk
-        @(SomeSecond BlockQuery blk)
-        codecConfig
-        blockVersion
-        (SomeSecond blockQuery)
-      GetPartialLedgerConfig ->
-        encodeTag 1
+      TopLevelQueryDisabled ->
```

Comment:
When `ToplevelQueryDisabled`, there are no top-level queries. That's why the previous code was correct and the new code here is contradictory.

Before we have any activated top-level queries, we'll have to introduce a `TopLevelQueryV_1` or some such in that enumeration type.